### ISSUE TITLE
bug:  failure to build youtube 19.03.35 with default settings

### ISSUE THREAD

**Issue by akirapink** [ReVanced/revanced-cli#312]
### Bug description

failure to build yt 19.03.35 with default settings on linux mint 21.3
command used:
`java -jar revanced-cli-4.4.0-all.jar   patch -b revanced-patches-4.0.2.jar   -m revanced-integrations-1.2.1.apk   -o out.apk base.apk`

### Error logs

```shell
akira@Lenovo-IdeaPad-S145-15API:~/Desktop$ java -jar revanced-cli-4.4.0-all.jar   patch -b revanced-patches-4.0.2.jar   -m revanced-integrations-1.2.1.apk   -o out.apk base.apk
INFO: Loading patches 
INFO: Decoding app manifest 
INFO: Predictive back gesture excluded 
INFO: Remove screenshot restriction excluded 
INFO: Spoof SIM country excluded 
INFO: Enable Android debugging excluded 
INFO: Override certificate pinning excluded 
INFO: Change package name excluded 
INFO: Export all activities excluded 
INFO: Spoof Wi-Fi connection excluded 
INFO: Remove screen capture restriction excluded 
INFO: Enable slide to seek excluded 
INFO: Remove player controls background excluded 
INFO: Change header excluded 
INFO: Custom branding excluded 
INFO: Setting patch options 
INFO: Merging integrations 
INFO: Deleting existing resource cache directory 
INFO: Decoding resources 
INFO: Executing patches 
INFO: Alternative thumbnails succeeded 
INFO: Always repeat succeeded 
INFO: Announcements succeeded 
INFO: Bypass URL redirects succeeded 
INFO: Change start page succeeded 
INFO: Client spoof succeeded 
INFO: Comments succeeded 
INFO: Copy video URL succeeded 
INFO: Custom player overlay opacity succeeded 
INFO: Disable auto captions succeeded 
INFO: Disable fullscreen ambient mode succeeded 
INFO: Disable player popup panels succeeded 
INFO: Disable precise seeking gesture succeeded 
INFO: Disable resuming Shorts on startup succeeded 
INFO: Disable rolling number animations succeeded 
INFO: Disable suggested video end screen succeeded 
INFO: Disable zoom haptics succeeded 
INFO: Enable debugging succeeded 
INFO: Enable tablet layout succeeded 
INFO: External downloads succeeded 
INFO: GmsCore support succeeded 
INFO: HDR auto brightness succeeded 
INFO: Hide 'Load more' button succeeded 
INFO: Hide Shorts components succeeded 
INFO: Hide ads succeeded 
INFO: Hide album cards succeeded 
INFO: Hide autoplay button succeeded 
INFO: Hide breaking news shelf succeeded 
INFO: Hide captions button succeeded 
INFO: Hide cast button succeeded 
INFO: Hide crowdfunding box succeeded 
INFO: Hide endscreen cards succeeded 
INFO: Hide filter bar succeeded 
INFO: Hide floating microphone button succeeded 
INFO: Hide info cards succeeded 
INFO: Hide layout components succeeded 
INFO: Hide player buttons succeeded 
INFO: Hide seekbar succeeded 
INFO: Hide timestamp succeeded 
INFO: Hide video action buttons succeeded 
INFO: Minimized playback succeeded 
INFO: Navigation buttons succeeded 
INFO: Open links externally succeeded 
INFO: Playback speed succeeded 
INFO: Player flyout menu succeeded 
INFO: Remember video quality succeeded 
INFO: Remove tracking query parameter succeeded 
INFO: Remove viewer discretion dialog succeeded 
INFO: Restore old seekbar thumbnails succeeded 
INFO: Restore old video quality menu succeeded 
INFO: Return YouTube Dislike succeeded 
INFO: Seekbar tapping succeeded 
INFO: SponsorBlock succeeded 
INFO: Spoof app version succeeded 
INFO: Spoof device dimensions succeeded 
INFO: Swipe controls succeeded 
INFO: Tablet mini player succeeded 
INFO: Theme succeeded 
INFO: Video ads succeeded 
INFO: Wide searchbar succeeded 
INFO: Change package name succeeded 
INFO: Compiling patched dex files 
INFO: Compiled classes.dex 
INFO: Compiled classes2.dex 
INFO: Compiled classes3.dex 
INFO: Compiled classes4.dex 
INFO: Compiled classes5.dex 
INFO: Compiled classes6.dex 
INFO: Compiled classes7.dex 
INFO: Compiled classes8.dex 
INFO: Compiling modified resources 
WARNING: /home/akira/Desktop/out-resource-cache/res/layout/revanced_settings_with_toolbar.xml:18: error: resource drawable/yt_outline_arrow_left_black_24 (aka app.revanced.android.youtube:drawable/yt_outline_arrow_left_black_24) not found. 
WARNING: error: failed linking file resources. 
brut.androlib.exceptions.AndrolibException: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_4512024981369675188576217754136429558.tmp, link, -o, /home/akira/Desktop/out-resource-cache/aapt_temp_file, --package-id, 127, --min-sdk-version, 26, --target-sdk-version, 34, --version-code, 1544019392, --version-name, 19.03.35, --no-auto-version, --no-version-vectors, --no-version-transitions, --no-resource-deduping, --allow-reserved-package-id, -e, /tmp/APKTOOL3426721170813695656.tmp, -0, arsc, -I, /home/akira/Desktop/out-resource-cache/1.apk, --manifest, /home/akira/Desktop/out-resource-cache/AndroidManifest.xml, /home/akira/Desktop/out-resource-cache/build/resources.zip]
	at brut.androlib.AaptInvoker.invokeAapt2(AaptInvoker.java:246)
	at brut.androlib.AaptInvoker.invokeAapt(AaptInvoker.java:391)
	at app.revanced.patcher.data.ResourceContext.get(ResourceContext.kt:128)
	at app.revanced.patcher.Patcher.get(Patcher.kt:265)
	at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:301)
	at picocli.CommandLine.executeUserObject(CommandLine.java:2026)
	at picocli.CommandLine.access$1500(CommandLine.java:148)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2453)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2415)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2417)
	at picocli.CommandLine.execute(CommandLine.java:2170)
	at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
Caused by: brut.common.BrutException: could not exec (exit code = 1): [/tmp/brut_util_Jar_4512024981369675188576217754136429558.tmp, link, -o, /home/akira/Desktop/out-resource-cache/aapt_temp_file, --package-id, 127, --min-sdk-version, 26, --target-sdk-version, 34, --version-code, 1544019392, --version-name, 19.03.35, --no-auto-version, --no-version-vectors, --no-version-transitions, --no-resource-deduping, --allow-reserved-package-id, -e, /tmp/APKTOOL3426721170813695656.tmp, -0, arsc, -I, /home/akira/Desktop/out-resource-cache/1.apk, --manifest, /home/akira/Desktop/out-resource-cache/AndroidManifest.xml, /home/akira/Desktop/out-resource-cache/build/resources.zip]
	at brut.util.OS.exec(OS.java:104)
	at brut.androlib.AaptInvoker.invokeAapt2(AaptInvoker.java:242)
	... 13 more
```

### Additional context

revanced manager also fails to patch on mobile. possibly unrelated

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by Ushie** [comment]
ReVanced doesn't support patching split APKs, please patch a universal APK

This can be obtained from sources like APKMirror

**Comment by akirapink** [comment]
sorry

**Comment by oSumAtrIX** [comment]
Following up, it does, but it being successful depends on AAPT2/ the decoding and encoding logic of resources
